jumeg/jumeg_interpolate_bads.py

```diff
@@ -52,9 +55,12 @@ def interpolate_bads(inst, reset_bads=True, mode='accurate', origin=None, verbos
     else:
         origin = origin
 
-    from mne.channels.interpolation import _interpolate_bads_eeg
+    if check_version('mne', '0.20'):
+        _interpolate_bads_eeg(inst, origin=origin)
+    else:
+        # workaround for compatibility with 0.19, should be removed soon
+        _interpolate_bads_eeg(inst)
```

Comment:
Should we add a warning or something that the interpolation for versions < 0.20 is different for EEG and MEG?

I almost preferred it the way it was before because then it crashes and it becomes obvious that something is wrong.

Reply:
That's right, it's best to add a warning.